docs/animations/Blur.md

```diff
@@ -119,7 +119,7 @@ Await MyUIElement.Blur(value:=5, duration:=2500, delay:=250).StartAsync()    ' B
 
 ## API
 
-* [Blur source code](https://github.com/Microsoft/WindowsCommunityToolkit//blob/master/Microsoft.Toolkit.Uwp.UI.Animations/Behaviors/Blur.cs)
+* [Blur source code](https://github.com/Microsoft/WindowsCommunityToolkit//blob/master/Microsoft.Toolkit.Uwp.UI.Media/Animations/BlurEffectAnimation.cs)

```

Comment:
There a a bunch of `//blob` with the double slash in the URL that you might want to find/replace in a batch

Reply:
Thanks, @nschonni should be good now. Please let me know if you find anything else that needs fixing.

Reply:
Nope, but you probably need to rebase this one to fix the merge conflicts